packages/gtk_doc.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gtk_doc < Package
+class Gtk_doc < Meson
   description 'Documentation tool for public library API'
   homepage 'https://www.gtk.org/gtk-doc/'
   version '1.33.2-1'
```

Comment:
Update to [1.34.0](https://download.gnome.org/sources/gtk-doc/1.34/gtk-doc-1.34.0.tar.xz)?